nvim/lua/core/plugins.lua

```diff
@@ -21,7 +21,7 @@ require("lazy").setup {
   { "nvim-lua/plenary.nvim" }, -- lua functions that many plugins use
 
   -- Colorscheme
-  { "catppuccin/nvim", name = "catppuccin" },
+  { "rebelot/kanagawa.nvim" },
```

Comment:
if you're applying the default setup config, then you can specify the plugin using
```lua
{ "rebelot/kanagawa.nvim", opts = {} },
```

the opts param will call the setup function for you